### PR TITLE
Fix restore

### PR DESCRIPTION
**Pull request by matkam**
Added LZO file decompression and updated table matcher string

### CODE REVIEW

File: cassandra_snapshotter/utils.py [hunk 1]
```diff
@@ -64,3 +71,39 @@ def map_wrap(f):
     def wrapper(*args, **kwargs):
         return apply(f, *args, **kwargs)
     return wrapper
+
+
+def check_lzop():
+    try:
+        subprocess.call([LZOP_BIN, '--version'])
+    except OSError:
+        print("{!s} not found on path".format(LZOP_BIN))
```
Comment (tbarbugli) [inline cassandra_snapshotter/utils.py]: what about we add a system.exit(1) if lzop is missing?

Comment (matkam) [inline cassandra_snapshotter/utils.py]: sure, sounds good.

File: cassandra_snapshotter/snapshotting.py [hunk 2]
```diff
@@ -189,7 +190,20 @@ def _download_key(self, key):
         filename = "./{!s}/{!s}/{!s}_{!s}".format(
             r.group(2), r.group(3),
             key.name.split('/')[2], key.name.split('/')[-1])
-        key.get_contents_to_filename(filename)
+
+        if filename.endswith('.lzo'):
+            filename = re.sub('\.lzo$', '', filename)
+            lzop_pipe = decompression_pipe(filename)
+            key.open_read()
+            for bytes in key:
```
Comment (tbarbugli) [inline cassandra_snapshotter/snapshotting.py]: `bytes` is a reserved keyword

**Comment by matkam** [comment]
Let me know if you want me to remove my formatting changes

**Comment by matkam** [comment]
:+1:

**Pull request merged**